ui/analyse/src/study/studyShare.ts

```diff
@@ -137,6 +137,21 @@ export function view(ctrl: StudyShareCtrl): VNode {
         },
         ctrl.trans.noarg(ctrl.relay ? 'downloadGame' : 'chapterPgn')
       ),
+      h(
+        'a.button.text',
+        {
+          attrs: {
+            'data-icon': '',
```

Comment:
```suggestion
            'data-icon': '',
```
[It now makes sense to use U+E070](https://github.com/lichess-org/lila/pull/11384) and the icon should be available after merging the changes from upstream's master branch